### PR TITLE
#757  test for fixed Xembly.escape() issue

### PR DESCRIPTION
**Pull request by dmzaytsev**
for #757
created the unit test for fixed Xembly.escape() issue

### CODE REVIEW

**Comment by karato** [comment]
@dmzaytsev I will find a reviewer for your pull requests shortly, thanks for contribution!

**Comment by karato** [comment]
@longtimeago do this review

File: netbout-web/src/test/java/com/netbout/rest/bout/XeMessageTest.java [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+import org.hamcrest.MatcherAssert;
+import org.hamcrest.Matchers;
+import org.junit.Test;
+import org.xembly.Xembler;
+
+/**
+ * Test case for {@link XeMessage}.
+ * @author Dmitry Zaytsev (dmitry.zaytsev@gmail.com)
+ * @version $Id$
+ * @since 2.16.1
```
Comment (longtimeago) [inline netbout-web/src/test/java/com/netbout/rest/bout/XeMessageTest.java]: @dmzaytsev should be 2.16.2

**Comment by longtimeago** [comment]
@dmzaytsev one small comment and we are good to go

**Comment by dmzaytsev** [comment]
@longtimeago you're right, done
please take a look

**Comment by longtimeago** [comment]
@rultor merge

**Comment by karato** [comment]
@ypshenychka please, review this ticket for compliance with our [QA rules](http://www.teamed.io/qa.html)

**Comment by ypshenychka** [comment]
@longtimeago 
According to our [QA Rules](http://www.teamed.io/qa.html):
>The code reviewer found at least three problems in the code.
Comments were mostly about design problems, not cosmetic issues.

Only one cosmetic issue was found during code review.

Please confirm that you will try to find at least three major problems next time.

**Comment by longtimeago** [comment]
@ypshenychka I confirm

**Comment by ypshenychka** [comment]
@longtimeago thank you.

**Comment by ypshenychka** [comment]
@karato Quality is acceptable.

**Comment by karato** [comment]
> @karato Quality is acceptable.

@ypshenychka thank you, let's try to make it "good" next time

**Comment by karato** [comment]
@longtimeago 10 mins was added to the account of @ypshenychka (for QA review), in transaction 63052940; Much obliged! I have added **18 mins** to your account in payment "63052958", 43 hours and 37 mins spent; you're getting extra minutes here (c=3); added +18 to your rating, now it is equal to [+3227](http://www.netbout.com/b/35024?open=rating)

**Pull request closed without merging**